src/lug/System/Unix/PathImplUnix.cpp

```diff
@@ -0,0 +1,47 @@
+#include <lug/System/Unix/PathImplUnix.hpp>

```

Comment:
I think that you can comment at the beginning of each function to indicate the attended path.
Like `/home/{LOGIN}/`